core/tests/protractor_utils/AdminPage.js

```diff
@@ -34,6 +34,58 @@ var AdminPage = function(){
   var roleSelect = element(by.css('.protractor-update-form-role-select'));
   var statusMessage = element(by.css('[ng-if="statusMessage"]'));
 
+  // The reload functions are used for mobile testing
+  // done via Browserstack. These functions may cause
+  // a problem when used to run tests directly on Travis.
```

Comment:
No action necessary here, but make sure you document this in your e2e docs!

Reply:
Note taken, thanks!